### PR TITLE
Update blink for new service calls

### PR DESCRIPTION
**Pull request by mkmer**
## Proposed change

Update service calls to match new keys, removing name and utilizing entity and config_entry values.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/105413
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/blink.markdown [hunk 1]
```diff
@@ -59,22 +59,26 @@ Any sequential calls to {% term services %}  relating to blink should have a min
 ### `blink.blink_update`
 
 Force a refresh of the Blink system.
+This service is depreciated, use `Home Assistant Core Integration: Update entity`
+| Service Data Attribute | Optional | Description                            |
+| ---------------------- | -------- | -------------------------------------- |
+| `config_entry_id`      | no       | Blink config to update.                |
```
Comment (frenck) [inline source/_integrations/blink.markdown]: We should remove deprecated services from our documentation.

File: source/_integrations/blink.markdown [hunk 2]
```diff
@@ -59,22 +59,26 @@ Any sequential calls to {% term services %}  relating to blink should have a min
...
 
 ### `blink.trigger_camera`
 
 Trigger a camera to take a new still image.
 
 | Service Data Attribute | Optional | Description                            |
 | ---------------------- | -------- | -------------------------------------- |
-| `entity_id`            | yes      | Camera entity to take picture with.    |
+| `entity_id`            | no.      | Camera entity to take picture with.    |
```
Comment (frenck) [inline source/_integrations/blink.markdown]: This change is unrelated, and also incorrect. This service seem to be an entity service, meaning it can be targeted without an entity ID: thus optional.

File: source/_integrations/blink.markdown [hunk 3]
```diff
@@ -59,22 +59,26 @@ Any sequential calls to {% term services %}  relating to blink should have a min
...
 
 ### `blink.save_video`
 
 Save the last recorded video of a camera to a local file. Note that in most cases, Home Assistant will need to know that the directory is writable via the `allowlist_external_dirs` in your `configuration.yaml` file (see example below).
 
 | Service Data Attribute | Optional | Description                              |
 | ---------------------- | -------- | ---------------------------------------- |
-| `name`                 | no       | Name of camera containing video to save. |
+| `entity_id`            | no       | Name of camera containing video to save. |
```
Comment (frenck) [inline source/_integrations/blink.markdown]: same

File: source/_integrations/blink.markdown [hunk 4]
```diff
@@ -90,6 +94,7 @@ Send a new pin to blink.  Since Blink's 2FA implementation is new and changing,
 
 | Service Data Attribute | Optional | Description                  |
 | ---------------------- | -------- | ---------------------------- |
+| `config_entry_id`      | no       | Blink config to send pin to. |
```
Comment (frenck) [inline source/_integrations/blink.markdown:91]: same

File: source/_integrations/blink.markdown [hunk 5]
```diff
@@ -173,7 +178,7 @@ The following example assumes your camera's name (in the Blink app) is `My Camer
   action:
     service: blink.save_video
     data:
-      name: "My Camera"
+      entity_id: "My Camera"
```
Comment (frenck) [inline source/_integrations/blink.markdown]: This seems wrong? Entity ID should not be part of the data but part of the target?

File: source/_integrations/blink.markdown [hunk 6]
```diff
@@ -200,6 +205,6 @@ The file name of the downloaded video file is not configurable.
   action:
     - service: blink.save_recent_clips
       data:
-        name: My Camera
+        entity_id: My Camera
```
Comment (frenck) [inline source/_integrations/blink.markdown]: same?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/blink.markdown (hunk 3 above)
Comment (mkmer) [reply source/_integrations/blink.markdown]: Thanks - I missed these after the recommended changes to the PR.

File: source/_integrations/blink.markdown (hunk 4 above)
Comment (mkmer) [reply source/_integrations/blink.markdown:91]: This one currently is not a target, we are only pointing at the config_entry via a selector.  Is there a different way to point at a config entry?

Comment (frenck) [reply source/_integrations/blink.markdown:91]: No, this was just my review mistake, sorry about that 👍

Comment (mkmer) [reply source/_integrations/blink.markdown:91]: I made plenty of those myself.  Learning new stuff everyday!

File: source/_integrations/blink.markdown [hunk 7]
```diff
@@ -54,27 +54,18 @@ Please note that each camera reports two different states: one as `sensor.blink_
 
 ## Services
 
-Any sequential calls to {% term services %}  relating to blink should have a minimum of a 5 second delay in between them to prevent the calls from being throttled and ignored.
-
-### `blink.blink_update`
-
-Force a refresh of the Blink system.
+Any sequential calls to {% term services %}  relating to blink should have a minimum of a 5 second delay in between them to prevent the calls from being throttled and ignored.  Services have targets.
```
Comment (MartinHjelmare) [inline source/_integrations/blink.markdown]:
```suggestion
Any sequential calls to {% term services %}  relating to blink should have a minimum of a 5 second delay in between them to prevent the calls from being throttled and ignored. The services that act on a camera needs a target parameter.
```

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]

File: source/_integrations/blink.markdown (hunk 6 above)
Comment (MartinHjelmare) [reply source/_integrations/blink.markdown]: "My Camera" isn't an entity_id.

File: source/_integrations/blink.markdown [hunk 8]
```diff
@@ -54,7 +54,7 @@ Please note that each camera reports two different states: one as `sensor.blink_
 
 ## Services
 
-Any sequential calls to {% term services %}  relating to blink should have a minimum of a 5 second delay in between them to prevent the calls from being throttled and ignored.  Services have targets.
+Any sequential calls to {% term services %}  relating to blink should have a minimum of a 5 second delay in between them to prevent the calls from being throttled and ignored.  The services that act on a camera needs a target parameter.
```
Comment (MartinHjelmare) [inline source/_integrations/blink.markdown]:
```suggestion
Any sequential calls to {% term services %}  relating to blink should have a minimum of a 5 second delay in between them to prevent the calls from being throttled and ignored. The services that act on a camera needs a target parameter.
```

File: source/_integrations/blink.markdown [hunk 9]
```diff
@@ -54,27 +54,18 @@ Please note that each camera reports two different states: one as `sensor.blink_
 
 ## Services
 
-Any sequential calls to {% term services %}  relating to blink should have a minimum of a 5 second delay in between them to prevent the calls from being throttled and ignored.
-
-### `blink.blink_update`
-
-Force a refresh of the Blink system.
+Any sequential calls to {% term services %}  relating to blink should have a minimum of a 5 second delay in between them to prevent the calls from being throttled and ignored. The services that act on a camera needs a target parameter.
```
Comment (MartinHjelmare) [inline source/_integrations/blink.markdown]:
```suggestion
Any sequential calls to {% term services %} relating to blink should have a minimum of a 5 second delay in between them to prevent the calls from being throttled and ignored. The services that act on a camera needs a target parameter.
```

File: source/_integrations/blink.markdown [hunk 10]
```diff
@@ -83,18 +74,26 @@ homeassistant:
...
+In addition to the services mentioned above, there are generic `camera`, `alarm_control_panel`, and `homeassistant` services are available. The `camera.enable_motion_detection` and `camera.disable_motion_detection` services allow for individual cameras to be enabled and disabled, respectively, within the Blink system. The `alarm_control_panel.alarm_arm_away` and `alarm_control_panel.alarm_disarm` services allow for the whole system to be armed and disarmed, respectively.  The `homeassistant.update_entity` service will force an update of the blink system.  Blink Mini cameras linked to an existing sync module cannot be armed/disarmed individually via Home Assistant.
```
Comment (MartinHjelmare) [inline source/_integrations/blink.markdown]:
```suggestion
In addition to the services mentioned above, there are generic `camera`, `alarm_control_panel`, and `homeassistant` services are available. The `camera.enable_motion_detection` and `camera.disable_motion_detection` services allow for individual cameras to be enabled and disabled, respectively, within the Blink system. The `alarm_control_panel.alarm_arm_away` and `alarm_control_panel.alarm_disarm` services allow for the whole system to be armed and disarmed, respectively. The `homeassistant.update_entity` service will force an update of the blink system.  Blink Mini cameras linked to an existing sync module cannot be armed/disarmed individually via Home Assistant.
```

File: source/_integrations/blink.markdown [hunk 11]
```diff
@@ -83,18 +74,26 @@ homeassistant:
...
+In addition to the services mentioned above, there are generic `camera`, `alarm_control_panel`, and `homeassistant` services are available. The `camera.enable_motion_detection` and `camera.disable_motion_detection` services allow for individual cameras to be enabled and disabled, respectively, within the Blink system. The `alarm_control_panel.alarm_arm_away` and `alarm_control_panel.alarm_disarm` services allow for the whole system to be armed and disarmed, respectively. The `homeassistant.update_entity` service will force an update of the blink system.  Blink Mini cameras linked to an existing sync module cannot be armed/disarmed individually via Home Assistant.
```
Comment (MartinHjelmare) [inline source/_integrations/blink.markdown]:
```suggestion
In addition to the services mentioned above, there are generic `camera`, `alarm_control_panel`, and `homeassistant` services available. The `camera.enable_motion_detection` and `camera.disable_motion_detection` services allow for individual cameras to be enabled and disabled, respectively, within the Blink system. The `alarm_control_panel.alarm_arm_away` and `alarm_control_panel.alarm_disarm` services allow for the whole system to be armed and disarmed, respectively. The `homeassistant.update_entity` service will force an update of the blink system. Blink Mini cameras linked to an existing sync module cannot be armed/disarmed individually via Home Assistant.
```

**Review by MartinHjelmare — APPROVED** [review]
Thanks!

Side note: We're still talking about `scan_interval` in the docs page as it was a config option. That sounds like it needs an update.

**Comment by mkmer** [comment]
> Thanks!
> 
> Side note: We're still talking about `scan_interval` in the docs page as it was a config option. That sounds like it needs an update.

This currently IS a configuration option in the config flow.  What is the current standing on this?  If a user has all wired based cameras they can poll much faster, but maybe they could just use the update service to speed things up?

**Comment by MartinHjelmare** [comment]
Since we're passing `force=True` when refreshing in the coordinator, the config option seems useless. It's also not in line with our guidelines to have that option.

**Comment by frenck** [comment]
I'm going to merge this in, as the release is today.

However, @mkmer, the scan_interval options must be removed, we don't allow these in the UI.

../Frenck

**Review by frenck — APPROVED** [review]

**Comment by mkmer** [comment]
There is a doc update attached to a PR removing the scan_interval.
https://github.com/home-assistant/home-assistant.io/pull/30575
